backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/controllers/ShopController.java

```diff
@@ -0,0 +1,37 @@
+/*Copyright 2020 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.*/
+
+package com.google.gpay.virtualqueue.backendservice.controllers;
+
+import com.google.gpay.virtualqueue.backendservice.entities.Shop;
+import com.google.gpay.virtualqueue.backendservice.service.ShopService;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@CrossOrigin
+public class ShopController {
+    @Autowired
+    public ShopService shopService;
+
+    @PostMapping("/addshop")
```

Comment:
I think we should name this just 'shop'.

Read up on API naming convention.

Reply:
Ok, I will look into that!

Reply:
Done!